lib/defaults.js

```diff
@@ -1,21 +1,14 @@
 'use strict';
-
-const controllers = require('hof-controllers');
-const path = require('path');
-
-const testPath = path.resolve(__dirname, '../test/');
-const realPath = path.dirname(module.parent.parent.filename);
 /* eslint no-process-env: "off" */
-const caller = process.env.NODE_ENV === 'test' ? testPath : realPath;
 
 const defaults = {
+  root: process.cwd(),
   translations: 'translations',
-  caller: caller,
   start: true,
   getCookies: true,
   getTerms: true,
   viewEngine: 'html',
-  baseController: controllers.base,
+  baseController: require('hof-controllers').base,
```

Comment:
Because it's a single-use variable. There didn't seem any value in the extra line of code as I don't think removing the variable impacted readability.